modules/docs/arrow-docs/docs/docs/effects/async/README.md

```diff
@@ -87,6 +87,18 @@ Behind the scenes `continueOn()` starts a new coroutine and passes the rest of t
 
 The function `continueOn()` is also available inside [`Monad Comprehensions`]({{ '/docs/patterns/monad_comprehensions' | relative_url }}).
 
+### mapEffect
+
+Helper function that provides an easy way to construct a suspend effect
+
+```kotlin
+val userCreation = IO.async().run {
+  just(id)
+    .mapEffect { id -> createUserFromId(id) }
+}
+userCreation.request(it)
+```
```

Comment:
This snippet is invalid because the type of `userCreation` is `Kind<ForIO, User>`.

Reply:
Let's remove it then as suggested.